DependencyInjection/DoctrineMigrationsExtension.php

```diff
@@ -23,6 +23,12 @@ public function load(array $configs, ContainerBuilder $container) : void
     {
         $configuration = new Configuration();
 
+        if ($container->hasParameter('kernel.root_dir')) {
+            $container->setParameter('doctrine.migrations.dir', '%kernel.root_dir%/DoctrineMigrations');
+        } else {
+            $container->setParameter('doctrine.migrations.dir', '%kernel.project_dir%/src/Migrations');
```

Comment:
Sorry for the late response.

As far as I can see this would still be a BC break for someone having an application that uses the old directory layout (i.e. they only updated their dependencies, but didn't adapt the directory structure established with Symfony Flex).

I think the only safe upgrade path is to deprecate not configuring this option and making it a mandatory one in the next major release.

Reply:
Indeed @xabbuh is right :cry: This will break for people with the old directory structure that just move to Symfony 5 without adapting to the new structure.

Reply:
> I think the only safe upgrade path is to deprecate not configuring this option and making it a mandatory one in the next major release.

I'll defer this to @goetas, since the next major release will change the config format anyways. We may just want to ignore this for now and fix it properly in 3.0.